src/test/java/io/stargate/sgv2/jsonapi/service/resolver/UpdateManyCommandResolverTest.java

```diff
@@ -227,77 +223,6 @@ public void dynamicFilterCondition() throws Exception {
               });
     }
 
-    @Test
-    public void dynamicFilterConditionSetVectorize() throws Exception {
```

Comment:
Instead of removing the test can we modify it to show how the data would look like? Will be helpful when changes are done around it.

Reply:
I have tests in the DocumentUpdaterTest class. Should be clear to see.
Reason of deleting this one, is because we no longer do updateClause vectorize.
Instead, we vectorize the document if found. Here in UpdateManyCommandResolverTest.java, we don't actually mimic document returned by DB.